utils/trytes.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#include "trytes.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define REQ_BODY                                                           \
```

Comment:
I am not sure if it makes to duplicate the request body template here.
Defer to @HowJMay

Reply:
Mark new issue #60 for this.